spec/jobs/job_configurations_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'job_configurations' do
+  describe 'weekly report' do
+    %w[drop_off_report authentication_report].each do |job_name|
+      it "schedules the #{job_name.humanize} to run after the end of the week with yesterday's date" do
+        report = GoodJob.configuration.cron[:"weekly_#{job_name}"]
+        expect(report[:class]).to eq("Reports::#{job_name.camelize}")
+
+        # Always passes the previous day as the argument
+        expect(report[:args].call).to eq([Time.zone.yesterday])
```

Comment:
let's add a `freeze_time` around these specs

Reply:
❓ @zachmargolis Do we do this automatically because of:

https://github.com/18F/identity-idp/blob/cff618ecb27545b0fa27e92ea96573b9f6886f7e/spec/rails_helper.rb#L134-L136

Reply:
That only applies to specs/groups with a `freeze_time: true` tag, I didn't see that here?